Utilities/sandbox/parse-manifest.sb

```diff
@@ -0,0 +1,19 @@
+(version 1)
+
+; Deny everything by default.
+(deny default)
+
+; Allow reading all files.
+(allow file-read*)
+
+; These are required by the Swift compiler.
+(allow process*)
+(allow sysctl*)
+(allow mach*)
+(allow signal)
+
+; Allow writing in temporary locations.
+(allow file-write*
+    (subpath "/private/var")
+    (subpath "/private/tmp")
+)
```

Comment:
If we think we can get away without it, maybe it would be safer to just remove it -- that way we won't unintentionally fail if it is required but only for some users; we will find it sooner.

I suspect the `/var` thing is for the module cache?